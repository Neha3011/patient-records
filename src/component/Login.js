import React from 'react';
import { Link } from 'react-router-dom';

/**
 * Describes the Login component
 * **/

class Login extends React.Component {
  render() {
    return (
      <div className="pharm__login">
        <Link to="patient">
          <button className="pharm__login__button pharm__login__button--patient">Login as Patient</button>
        </Link>
        <Link to="doctor">
          <button className="pharm__login__button pharm__login__button--doctor">Login as Doctor</button>
        </Link>
        <Link to="pharmacist">
          <button className="pharm__login__button pharm__login__button--pharmacist">Login as Pharmacist</button>
        </Link>
      </div>
    );
  }
}

export default Login;
