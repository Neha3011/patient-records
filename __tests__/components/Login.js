import React from 'react';
import Login from '../../src/component/Login';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';

test('render Header:', () => {
  const component = renderer.create(
      <Router>
        <Login />
      </Router>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});