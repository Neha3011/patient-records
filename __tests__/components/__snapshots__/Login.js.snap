// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Header: 1`] = `
<div
  className="pharm__login"
>
  <a
    href="#/patient"
    onClick={[Function]}
  >
    <button
      className="pharm__login__button pharm__login__button--patient"
    >
      Login as Patient
    </button>
  </a>
  <a
    href="#/doctor"
    onClick={[Function]}
  >
    <button
      className="pharm__login__button pharm__login__button--doctor"
    >
      Login as Doctor
    </button>
  </a>
  <a
    href="#/pharmacist"
    onClick={[Function]}
  >
    <button
      className="pharm__login__button pharm__login__button--pharmacist"
    >
      Login as Pharmacist
    </button>
  </a>
</div>
`;
